{"name": "Логин курьером с несуществующими данными", "status": "passed", "description": "Проверяем, что нельзя залогиниться несуществующими курьером, допустим, в логине, и что приходит ответ 404 с соответствующим письменным уведомлением", "steps": [{"name": "Создание рандомных логина, пароля и имени", "status": "passed", "start": 1715428323616, "stop": 1715428323616}], "start": 1715428323615, "stop": 1715428323976, "uuid": "749ca07d-8234-4ee5-9924-3a52cab043f5", "historyId": "66785b1c126aadbdccd73d6a3ddc1fcf", "testCaseId": "66785b1c126aadbdccd73d6a3ddc1fcf", "fullName": "tests.test_login_courier.TestLoginCourier#test_login_without_registration", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "LAPTOP-SSVUBLG9"}, {"name": "thread", "value": "5684-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}